src/frontend/interfaces/property-json/property-json.interface.ts

```diff
@@ -47,6 +47,8 @@ export interface PropertyJSON {
    * Property label
    */
   label: string;
+
+  description?: string;
```

Comment:
Can you add jsdoc comment for this attribute?

Reply:
Added, sorry I missed that.